packages/client/hmi-client/src/types/common.ts

```diff
@@ -236,3 +236,19 @@ export const programmingLanguageOptions = (): { name: string; value: string }[]
 				lang && `${lang[0].toUpperCase() + lang.slice(1)} (${ProgrammingLanguageVersion[lang]})`,
 			value: ProgrammingLanguageVersion[lang]
 		}));
+
+export enum DistributionType {
+	Constant = 'Constant',
+	Uniform = 'StandardUniform1'
+}
+
+export const DistributionTypeLabel: { [key in DistributionType]: string } = {
+	[DistributionType.Constant]: 'Constant',
+	[DistributionType.Uniform]: 'Uniform'
+};
+
+export const distributionTypeOptions = (): { name: string; value: string }[] =>
+	Object.values(DistributionType).map((dist) => ({
+		name: DistributionTypeLabel[dist],
+		value: dist
+	}));
```

Comment:
May want to consider having these distribution stuff in a separate file, as none of these are strictly speaking, type declarations anyway.

Reply:
put it in a file called distribution under `services/distribution.ts`